docs/slides/population_structure/index.qmd

```diff
@@ -14,157 +19,786 @@ format:
+the dataset, with respect to a panel of other World populations using
+a Mbuti as an outgroup population.
+
+```{r}
+#| eval: false
+#| class-source: my_code_size_abitsmaller
+#| classes: my_code_size_abitsmaller
+#| fig-width: 10
+#| fig-height: 6
+#| tidy: true
+#| tidy.opts: { width.cutoff: 78 }
+
+library("admixtools")
+library("tidyverse")
+
+admixtools::extract_f2('AADR', outdir='f2_AADR')
+
+f2_aadr <- read_f2("f2_AADR")
+fst_aadr <- fst(f2_aadr)
+fst_aadr
+
+f3_aadr <- f3(f2_aadr, pop1='Mbuti', pop2='Unknown', pop3=unique(ind$pop)) %>% arrange(est)
+f3_aadr
+
+
+viz_outgroup_f3 <- function(f3_res){
+  f3_res %>% mutate(pop3=fct_reorder(pop3, est)) %>%
+    ggplot(aes(x=pop3, y=est, ymin=est-3*se, ymax = est+3*se)) +
+    geom_point() + geom_errorbar() +
+    coord_flip() + theme_bw(25) + xlab(NULL) + ylab("f3")
+}
+viz_outgroup_f3(f3_aadr)
+```
+
+**Can you guess what was the Unknown population?**
+
+:::
+
+::: {.column width="47%"}
+![](assets/f3.png)
+:::
+::::
+
+## Relation between F-statistics and PCA
+
+![](assets/F_stats_PCA.png){fig-align="center"}
+
+[@Peter_2022]
+
+## F4-statistic, D-statistic: ABBA-BABA test
+
+:::: {.columns}
+::: {.column width="70%"}
+![](assets/D_stat_tree.png){fig-align="center"}
+:::
+
+::: {.column width="30%"}
+
+- Four populations considered: Ape is an outgroup, Europeans and
+  Africans are tested for their gene flow with Neanderthals
+
+- A – ancestral allele, B – derived allele
+
+- Count number of sites corresponding to ABBA and BABA situations
+
+$$D=\frac{N_{ABBA}-N_{BABA}}{N_{ABBA}+N_{BABA}}$$
+
+- $N_{ABBA}$ the total counts of ABBA patterns, $N_{BABA}$ the total
+  counts of BABA patterns
+
+- Exess of shared derived alleles between Europeans and Neanderthals
+  indicates a gene flow between them
+
+- Positive values of D imply Neanderthals are closer to Europeans
+
+- Negative values of D imply Neanderthals are closer to Africans
+
+:::
+::::
+
+## Example of D-statistic computation from 1000G project
+
+:::: {.columns}
+::: {.column width="50%"}
+
+We will use ANGSD [@Korneliussen_2014] and compute D-statistic in
+irder to test potential Neanderthal introgression to European (CEU)
+and African (YRI) populations from the 1000G project.
+
+```{bash}
+#| eval: false
+#| class-source: my_code_size_abitsmaller
+#| classes: my_code_size_abitsmaller
+#| fig-width: 10
+#| fig-height: 6
+#| tidy: true
+#| tidy.opts: { width.cutoff: 78 }
+
+echo Neandertal.bam > bams
+find $PWD -name '*YRI*.bam' | head -10 >> bams
+find $PWD -name '*CEU*.bam' | head -10 >> bams
+angsd -out out -doAbbababa 1 -bam bams -doCounts 1 -anc anc.fa
+Rscript jackKnife.R file=out.abbababa indNames=bams outfile=out_abbababa_results
+```
+
+and, finally, we will display the resulting table of D-statistics and
+plot the ABBA and BABA counts using custom R scripts:
+
+```{r}
+#| eval: true
+#| echo: false
+path<-"/home/nikolay/Documents/Teaching/PopGen_2023/data/"
```

Comment:
Same as above; you would need to copy-paste the output, and also save an image of the boxplot.

Reply:
@percyfal got it, will remove my local paths from the slides and insert the outputs as figures or text as you suggested above